.travis/run.sh

```diff
@@ -2,7 +2,7 @@
 
 set -o errexit
 set -o nounset
-set -o verbose
+set -o xtrace
```

Comment:
Makes it slightly easier to pick out the debug content in `xtrace` (prefixed with a `+`) compared to `verbose`.